subgraphs/compound-forks/schema.graphql

```diff
@@ -1074,3 +1074,16 @@ type _CircularBuffer @entity {
   " The current calculated number of blocks per day based on calculated block speed "
   blocksPerDay: BigDecimal!
 }
+
+# Helper entity for determining market.isActive status (dforce)
+# market.isActive = false if either mintPaused, redeemPaused, transferPaused = true
+type _MarketStatus @entity {
```

Comment:
given this is dforce-specific can you name it `_DforceMarketStatus`?

Reply:
fixed